### ISSUE TITLE
Space info on dashboard widget

### ISSUE THREAD

**Issue by buddh4** [nadilhassan/newsmodule#7]
Just a suggestion, but I think it would be good if there is a link and perhaps image of the space the news was posted in the dashboard widget.

**Comment by nadilhassan** [comment]
Yes, That's nice idea, Thank you . Btw what about dates of publishing the module to the humhub module marketplace ?
